app/views/categories/not_found.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.status 404
```

Comment:
We need to have agreement, how we will return errors for all cases and use them everywhere.